lib/search.js

```diff
@@ -8,6 +8,7 @@ const { BASE_URL } = require('./constants');
 
 const body = 'f.req=%5B%5B%5B%22qnKhOb%22%2C%22%5B%5Bnull%2C%5B%5B10%2C%5B10%2C50%5D%5D%2Ctrue%2Cnull%2C%5B96%2C27%2C4%2C8%2C57%2C30%2C110%2C79%2C11%2C16%2C49%2C1%2C3%2C9%2C12%2C104%2C55%2C56%2C51%2C10%2C34%2C77%5D%5D%2Cnull%2C%5C%22%token%%5C%22%5D%5D%22%2Cnull%2C%22generic%22%5D%5D%5D';
 
+const SPECIAL_COUNTRIES = ['kr', 'hk', 'tw'];
```

Comment:
Can you use "korea" instead of "special"? I know it's not only Korea but still special is too vague to know when reading the code

Reply:
Maybe `korea_store`